packages/cli/src/util/dev/builder.ts

```diff
@@ -337,7 +337,11 @@ export async function executeBuild(
   // Enforce the lambda zip size soft watermark
   const maxLambdaBytes = bytes('50mb');
   for (const asset of Object.values(result.output)) {
-    if (asset.type === 'Lambda') {
+    if (
+      asset.type === 'Lambda' &&
+      typeof asset.runtime === 'string' &&
+      asset.runtime.startsWith('python')
```

Comment:
@lucleray does this not do the opposite?

Doesn't this effectively _only_ apply the size limit to lambdas whose runtime start with "python"?